src/components/pill-selector-dropdown/PillSelector.js

```diff
@@ -227,6 +236,9 @@ class PillSelector extends React.Component<Props, State> {
                         suggestedPillsData={suggestedPillsData}
                         title={suggestedPillsTitle}
                     />
+                    <span id={this.errorMessageID} className="accessibility-hidden" role="alert">
```

Comment:
This will allow the error message to get read with a screen reader (in many cases, though unfortunately not all cases... but better than current anyway).